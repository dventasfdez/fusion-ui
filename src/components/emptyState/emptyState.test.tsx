import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import EmptyState, {EmptyStateTitle, EmptyStateDescription, EmptyStateImg} from './emptyState';

const ESTest: React.FC<any> = () => (
  <EmptyState>
    <EmptyStateImg src="https://picsum.photos/150" />
    <EmptyStateTitle>Something is wrong </EmptyStateTitle>
    <EmptyStateDescription>
      We are unable to connect to the service. Click on Retry to try again or View Log to learn more the issue{' '}
    </EmptyStateDescription>
    <button>retry</button>
    <a>View Log</a>
  </EmptyState>
);

test('Component renders and match snapshot', () => {
  const component = renderer.create(<ESTest />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
