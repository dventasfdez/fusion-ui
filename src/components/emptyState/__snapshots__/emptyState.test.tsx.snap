// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders and match snapshot 1`] = `
<div
  className="empty-state "
>
  <img
    src="https://picsum.photos/150"
  />
  <h4
    className="empty-state-title "
  >
    Something is wrong 
  </h4>
  <div
    className="subtitle "
  >
    We are unable to connect to the service. Click on Retry to try again or View Log to learn more the issue
     
  </div>
  <button>
    retry
  </button>
  <a>
    View Log
  </a>
</div>
`;
